knora-ontologies/knora-base.ttl

```diff
@@ -724,9 +735,7 @@
 
              :subjectClassConstraint :User ;
 
-             :objectClassConstraint :knoraProject ;
-
-             rdfs:subPropertyOf foaf:member .
```

Comment:
Yes, as the pull request description says: "Don't use `foaf:member`, because it means the opposite of what was intended."